### PR TITLE
Fix variables in 3dvar_gfs_aero.yaml

### PR DESCRIPTION
**Pull request by andytangborn**
Correct variable names in 3dvar_gfs_aero.yaml and change output to analysis increment in model (tiles) space.

### CODE REVIEW

File: parm/aero/variational/3dvar_gfs_aero.yaml [hunk 1]
```diff
@@ -2,7 +2,9 @@ cost function:
   cost type: 3D-Var
   window begin: '$(AERO_WINDOW_BEGIN)'
   window length: $(AERO_WINDOW_LENGTH)
-  analysis variables: &3dvars [ua,va,t,ps,sphum,ice_wat,liq_wat,o3mr]
+  analysis variables: &3dvars [sulf,bc1,bc2,oc1,oc2,
```
Comment (CoryMartin-NOAA) [inline parm/aero/variational/3dvar_gfs_aero.yaml:5]: We need to talk about this offline, I fear we have work ahead of us in UFO/FV3-JEDI to support the UFS coupled model with aerosols. So let's hold off on merging this until I know for sure.

Comment (andytangborn) [reply parm/aero/variational/3dvar_gfs_aero.yaml:5]: I pushed a new commit that includes the analysis increment that we tested in the workflow, so this PR both corrects the variable list and changes the analysis output to increment.

Comment (CoryMartin-NOAA) [reply parm/aero/variational/3dvar_gfs_aero.yaml:5]: Does `sulf` work? Or does it fail on that?

Comment (CoryMartin-NOAA) [reply parm/aero/variational/3dvar_gfs_aero.yaml:5]: Also it looks like this PR does *not* add in increment write...

Comment (CoryMartin-NOAA) [reply parm/aero/variational/3dvar_gfs_aero.yaml:5]: @andytangborn I just realized that you did not push a new commit to this PR... it is from 10 days ago.

**Comment by andytangborn** [comment]
Sulf will fail. When I ran it I used dms.

**Comment by andytangborn** [comment]
Changing to a different branch.

**Comment by andytangborn** [comment]
OK. Let me see what happened.

**Pull request closed without merging**